README.md

```diff
@@ -31,9 +31,16 @@ Please check [VDVS Installation User Guide](http://vmware.github.io/vsphere-stor
 
 ## Supported Platforms
 
-**ESXi:** 6.0 and above<br />
-**Docker (Linux):** 1.12 and higher (Recommended 1.13/17.03 and above to use managed plugin)<br />
-**Docker (Windows):** 1.13/17.03 and above (Windows containers mode only)
+**ESXi:** 6.0U2 and above<br />
+**Docker (Linux):** 17.06.1 and above to use managed plugin (v17.06.1 is having an important fix for data loss issue)<br />
+**Docker (Windows):** 17.06 and above (Windows containers mode only)<br />
+**Guest Operating System**:
+- Ubuntu 14.04 or higher (64 bit)
+   - Needs Upstart or systemctl to start and stop the service
+   - Needs [open vm tools or VMware Tools installed](https://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=340) ```sudo apt-get install open-vm-tools```
+- RedHat and CentOS
```

Comment:
Please split RedHat and CentOS on different lines.
Please mention the specific RHEL & CentOS versions that are validated.

Reply:
removed `CentOS`  part and added specific version for RHEL